editor/src/core/model/project-import.ts

```diff
@@ -132,31 +129,25 @@ export async function importZippedGitProject(
           )
           break
         }
-        case 'UI_JS_FILE':
-        case 'CODE_FILE':
+        case 'TEXT_FILE':
           const loadedFile = await attemptedTextFileLoad(shiftedFileName, file)
           if (loadedFile == null) {
             errors.push(`Unable to parse file ${shiftedFileName} as a text file`)
           } else {
-            if (expectedFileType === 'UI_JS_FILE') {
-              // TODO We really need a way to represent unparsed files, or a way to separate the parsed file from the contents
-              loadedProject = addFileToProjectContents(
-                loadedProject,
-                shiftedFileName,
-                uiJsFile(
-                  left(parseFailure(null, null, null, [], loadedFile)),
-                  null,
+            // TODO We really need a way to represent unparsed files, or a way to separate the parsed file from the contents
+            loadedProject = addFileToProjectContents(
+              loadedProject,
+              shiftedFileName,
+              textFile(
+                textFileContents(
+                  loadedFile,
+                  parseFailure(null, null, null, []),
```

Comment:
Now this can become `unparsed`, and we could probably update the revision state to `CodeAhead`

Reply:
Done in adc65a8